test/integration/controller/jobset_controller_test.go

```diff
@@ -354,6 +355,20 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				},
 			},
 		}),
+		ginkgo.Entry("suspended jobset will not start", &testCase{
```

Comment:
I need to add these three cases here:  

1) Start a JobSet as suspend should have suspend jobs.
2) Resuming a suspend job should start it.
3) Starting a job then suspended should suspend.

Reply:
Added.